visualization/.eslintrc.js

```diff
@@ -14,48 +20,24 @@ module.exports = {
 		"@typescript-eslint"
 	],
 	"rules": {
-		"@typescript-eslint/class-name-casing": "error",
-		"@typescript-eslint/explicit-member-accessibility": [
-			"error",
-			{
-				"accessibility": "explicit",
-				"overrides": {
-					"constructors": "no-public"
-				}
-			}
-		],
-		"@typescript-eslint/triple-slash-reference": "error",
-		"curly": "error",
-		"new-parens": "error",
-		"no-caller": "error",
-		"no-cond-assign": "error",
-		"no-console": [
-			"warn",
-			{
-				"allow": [
-					"warn",
-					"dir",
-					"timeLog",
-					"assert",
-					"clear",
-					"count",
-					"countReset",
-					"group",
-					"groupEnd",
-					"table",
-					"dirxml",
-					"error",
-					"groupCollapsed",
-					"Console",
-					"profile",
-					"profileEnd",
-					"timeStamp",
-					"context"
-				]
-			}
-		],
-		"no-empty": "error",
-		"no-eval": "error",
-		"no-var": "error"
+		// Do not apply inappropriate rules below
+		"@typescript-eslint/no-inferrable-types": "off",
```

Comment:
why shouldnt we apply this?

Reply:
The rule removes primitive types for variable declarations. Because I don’t want eslint to remove our types. Now after writing some JavaScript again, in my opinion there is nothing more helpful than types. So please let’s keep them.

Reply:
Yeah, but especially on primitive types I don't see a reason to use them.

Do you have an example in our codebase where this is useful?

Reply:
here you can see the [changes](https://github.com/MaibornWolff/codecharta/compare/tech/878/eslint-recommended-ruleset...spike/remove-primitive-types), that will be made when enabling that rule